modules/@shopify/checkout-sheet-kit/src/pixels.d.ts

```diff
@@ -408,16 +465,45 @@ interface CheckoutLineItem {
   /* The discounts that have been applied to the checkout line item by a */
   /* discount application. */
   discountAllocations?: DiscountAllocation[];
+  /* The combined price of all of the items in the line item after line-level discounts have been applied. */
+  finalLinePrice?: MoneyV2;
   /* A globally unique identifier. */
   id?: string;
+  /** The properties of the line item. A shop may add, or enable customers to add custom information to a line item.
+   * Line item properties consist of a key and value pair.
+   */
+  properties?: Property[];
   /* The quantity of the line item. */
   quantity?: number;
   /* The title of the line item. Defaults to the product's title. */
+  sellingPlanAllocation?: SellingPlanAllocation;
   title?: string;
   /* Product variant of the line item. */
   variant?: ProductVariant;
 }
 
+interface SellingPlanAllocation {
+  /**
+   * A representation of how products and variants can be sold and purchased. For example, an individual selling plan could be
+   * '6 weeks of prepaid granola, delivered weekly'.
+   */
+  sellingPlan?: SellingPlan;
+}
+
+interface SellingPlan {
+  /* A globally unique identifier. */
+  id?: string;
+  /* The name of the selling plan. For example, '6 weeks of prepaid granola, delivered weekly'. */
+  name?: string;
+}
+
+interface Property {
+  /* The key for the property. */
+  key?: string;
+  /* The value for the property. */
+  value?: string;
+}
```

Comment:
Mind maintaining the alphabetical ordering in this file?

Reply:
I don't think much of it is alphabetical, I'll reorder the ones I added though